### ISSUE TITLE
Service is created for every child type when using `Scope.REQUEST`

### ISSUE THREAD

**Issue by iagomelanias** [nestjs/graphql#181]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Injecting a service using the scope `REQUEST` is creating the service for every child type of a query.

## Expected behavior
The service should only be created once.

## Minimal reproduction of the problem with instructions
In the following query, we try to get all the posts of a specific user.
```gql
query {
  author(id: 1) {
    id
    email
    name
    posts {
      id
      title
      tags
      createdAt
    }
  }
}
```

The field tags is resolved dynamically using the feature `@ResolveProperty()`.

```ts
@Resolver('Post')
export class PostResolvers {
  constructor(private myService: MyService) {}

  @ResolveProperty()
  tags() {
    // the logic here
  }
}
```

`MyService` is going to be created for every post published by the author instead of only one time. Since `Scope.REQUEST` was designed to create the service for each request, it looks like a bug,

## What is the motivation / use case for changing the behavior?
By using `Scope.REQUEST`, we can create a [DataLoader](https://github.com/graphql/dataloader) for every request.

## Environment

<pre><code>
Nest version: 6.0.4

 
For Tooling issues:
- Node version: 11.8.0
- Platform:  Mac

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Thanks for reporting. It should be fixed in `@nestjs/graphql@6.0.4` :)

**Comment by iagomelanias** [comment]
Amazing! Thank you 🤗

**Comment by iagomelanias** [comment]
It seems that `6.0.4` was not published to NPM. 👀

**Comment by kamilmysliwiec** [comment]
Oh thanks @iagomelanias, fixed!

**Comment by iagomelanias** [comment]
I can confirm it was fixed in `6.0.4`. Thank you! 🙌
